logic/suggestions/tossStats_test.go

```diff
@@ -0,0 +1,32 @@
+package suggestions
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTossStatsCalculate(t *testing.T) {
+	ts := &tossStats{
+		allPts: []int{
+			1, 1, 1,
+			4, 4, 4,
+			10, 10,
+		},
+	}
+
+	ts.add(10)
+	assert.Len(t, ts.allPts, 9)
+
+	// before calling calculate
+	assert.Zero(t, ts.Min())
+	assert.Zero(t, ts.Median())
+	assert.Zero(t, ts.Avg())
+	assert.Zero(t, ts.Max())
+
+	ts.calculate()
+	assert.Equal(t, 1, ts.Min())
+	assert.Equal(t, float64(4), ts.Median())
+	assert.Equal(t, float64(5), ts.Avg())
```

Comment:
It's not the best practice to compare floats for equality (because they can be off by minute differences). It works here because you're using integers, but what if you tried to compute an average of `[1, 1, 1]`?

`assert.InEpsilon` is meant for this use case.